content/docs/ops/runbook/troubleshooting-kubernetes.md

```diff
@@ -25,6 +28,20 @@ The `Events` section should indicate why the pod cannot be started.  Resolve the
 /var/vcap/packages/kubernetes/bin/kubectl --namespace :namespace delete pod :pod-name
 ```
 
+#### Manually pulling an image from Docker
+
+When you are publishing a new Docker image and not updating the tag for it,
```

Comment:
This could be more specific--something like

> By default, kubernetes does not pull docker images that already exist on the node. If you update an existing image and tag, you can force kubernetes to pull the latest version from a jumpbox:

```shell
bosh -d kubernetes ssh minion 'bash -c "/var/vcap/packages/docker/bin/docker --host unix:///var/vcap/sys/run/docker/docker.sock pull ${DOCKER_USER}/${IMAGE_NAME}:${DOCKER_TAG}"'
```